sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/AnalyzeSentimentResult.java

```diff
@@ -30,6 +30,9 @@ public AnalyzeSentimentResult(String id, TextDocumentStatistics textDocumentStat
      * Get the document sentiment.
      *
      * @return The document sentiment.
+     *
+     * @throws TextAnalyticsException if result has {@code isError} equals to true and when a non-error property
```

Comment:
was this true before and we just forgot to add it in the docstring?

Reply:
throwExceptionIfError() throws TextAnalyticsException . we forget to add it in the docstring